modules/ocf_desktop/files/xsession/notify

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+import datetime
+import sched
+
+from gi.repository import Notify
+from ocflib.lab.stats import staff_in_lab
+from ocflib.misc.whoami import current_user
+from ocflib.printing.quota import get_connection
+from ocflib.printing.quota import get_quota
+
+
+NOTIFICATION_ICON = '/opt/share/xsession/images/ocf-color.png'
+
+# Format taken from paper command
+QUOTA_FORMAT = """\
+    → {semesterly} remaining this semester
+    → {daily} remaining today"""
+
+
+old_quota = None
+
+
+def show_quota_loop(scheduler):
+    global old_quota
+
+    user = current_user()
+    with get_connection() as c:
+        quota = get_quota(c, user)
+
+    if quota != old_quota:
+        Notify.Notification.new(
+            '{user} printing quota (as of {time})'.format(
+                user=user,
+                time=datetime.datetime.now().strftime('%-I:%M%P')
+            ),
+            QUOTA_FORMAT.format(semesterly=quota.semesterly, daily=quota.daily),
+            NOTIFICATION_ICON,
+        ).show()
+        old_quota = quota
+
+    # Schedule self to repeat next minute
+    scheduler.enter(60, 0, show_quota_loop, argument=(scheduler,))
+    scheduler.run()
```

Comment:
It would be better if it's running at a set time, since otherwise the time will drift by however long it takes to execute each time, but this is a pretty short script so that shouldn't matter. Thankfully timing definitely doesn't matter much for this script...